core/lib/dal/src/fri_prover_dal.rs

```diff
@@ -341,6 +341,30 @@ impl FriProverDal<'_, '_> {
         }
     }
 
+    pub async fn min_unproved_l1_batch_number_for_aggregation_round(
+        &mut self,
+        aggregation_round: AggregationRound,
+    ) -> Option<L1BatchNumber> {
+        {
```

Comment:
This additional scope is not necessary; it's present for some existing methods as a refactoring artifact. Also, it could make sense to use a consistent literal style for queries (e.g., ordinary literals with escaped newlines, like in `get_oldest_not_compressed_batch` – it is resistant to changes in the surrounding indentation and results in a searchable query in `sqlx-data.json`). OTOH, after #533 is merged, it would be possible to format queries automatically.